### PR TITLE
Add cromwell version 84 to table for 1.14

### PR DESCRIPTION
**Pull request by kathy-t**
https://ucsc-cgl.atlassian.net/browse/SEAB-4980

Adds a new row indicating that 1.14 uses cromwell version 84

### CODE REVIEW

**Review by svonworl — APPROVED** [review]

**Comment by denis-yuen** [comment]
@aofarrel has worked/fought with the link check recently. 
That said, 
```
(             faq: line  174) redirect  https://help.github.com/en/articles/approving-oauth-apps-for-your-organization/ - permanently to https://docs.github.com/en/organizations/managing-oauth-access-to-your-organizations-data/approving-oauth-apps-for-your-organization
(             faq: line  169) redirect  https://help.github.com/en/articles/enabling-oauth-app-access-restrictions-for-your-organization/ - permanently to https://docs.github.com/en/organizations/managing-oauth-access-to-your-organizations-data/enabling-oauth-app-access-restrictions-for-your-organization
...
(advanced-topics/best-practices/best-practices-secure-fair-workflows: line   88) redirect  https://www.openshift.com/blog/mitigate-impact-of-docker-hub-pull-request-limits - with Found to https://cloud.redhat.com/blog/mitigate-impact-of-docker-hub-pull-request-limits
```

If these redirects are legit and/or causing the failure, it would be good to update them

**Review by denis-yuen — APPROVED** [review]

**Comment by kathy-t** [comment]
> ```
> (             faq: line  174) redirect  https://help.github.com/en/articles/approving-oauth-apps-for-your-organization/ - permanently to https://docs.github.com/en/organizations/managing-oauth-access-to-your-organizations-data/approving-oauth-apps-for-your-organization
> (             faq: line  169) redirect  https://help.github.com/en/articles/enabling-oauth-app-access-restrictions-for-your-organization/ - permanently to https://docs.github.com/en/organizations/managing-oauth-access-to-your-organizations-data/enabling-oauth-app-access-restrictions-for-your-organization
> ...
> (advanced-topics/best-practices/best-practices-secure-fair-workflows: line   88) redirect  https://www.openshift.com/blog/mitigate-impact-of-docker-hub-pull-request-limits - with Found to https://cloud.redhat.com/blog/mitigate-impact-of-docker-hub-pull-request-limits
> ```
> 
> If these redirects are legit and/or causing the failure, it would be good to update them

Hmm, I don't think they're causing the failure. The failure is this link, but the link works on my browser so I'm not sure what the problem is

```
(      dictionary: line  198) broken    https://biodatacatalyst.nhlbi.nih.gov/ - HTTPSConnectionPool(host='biodatacatalyst.nhlbi.nih.gov', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))
```

**Comment by denis-yuen** [comment]
Oh, I think @aofarrel was looking into this, It looked like a known problem with the library that Sphinx uses for link checking. THink I'll override here since it's literally one line

**Comment by aofarrel** [comment]
@denis-yuen @kathy-t FYI, here's some relevant information regarding link checking being a bit unreliable:
* GitHub help pages: https://github.com/sphinx-doc/sphinx/issues/10343
  * This might apply to other websites too -- something about blocking certain user agents? 
* Files created by Sphinx itself: https://github.com/sphinx-doc/sphinx/issues/9383
* Zendesk, including Terra Support and OICR Support: https://wordpress.org/support/topic/zendesk-cloudflare-link-check-eror/
   * See also this other link checker _(that we do not use, just to be clear)_  but seems to suffer from the same problem: https://github.com/tcort/markdown-link-check/issues/109
   * TL;DR - Basically all link checkers seem to struggle with certain Cloudflare configurations
* doi.org: This might an issue on our end. Someone else claims they were having issues with it but later said oops, so we may want to investigate those URLs a little deeper: https://github.com/sphinx-doc/sphinx/issues/10915

Previously, Sphinx's link checker had issues with self-signed certificates. Those issues seem to have been resolved.

**Comment by aofarrel** [comment]
See also https://www.ssllabs.com/ssltest/analyze.html?d=biodatacatalyst.nhlbi.nih.gov&s=13.35.125.17&latest (courtesy of Denis on Slack)
